client/services/apiService.ts

```diff
@@ -1,20 +1,31 @@
 import { ProfileType, StoryType, UserType } from '../types';
 import axios, { AxiosResponse, isCancel, AxiosError } from 'axios';
 
-const BASE_URL = 'http://localhost:3000'; //this may need to change to IP address
+//const BASE_URL = 'http://localhost:3000'; //this may need to change to IP address
+const BASE_URL = 'http://10.0.2.2:3000'; //this is the URL used for android simulator
 
 export const getUser = async (id: number): Promise<UserType> => {
   try {
     const response: AxiosResponse<UserType> = await axios.get(`${BASE_URL}/users/${id}`);
-    console.log('getting user:', response.data);
+    
+    // console.log('getting user:', response.data);
     return response.data;
   } catch (e) {
     console.log('error getting user', e);
     throw e;
   }
 };
 
-// export const getAllProfiles = async (userId: number): Promise<ProfileType[]> => {};
+export const getAllProfiles = async (userId: number): Promise<ProfileType[]> => {
+  try {
+    const response: AxiosResponse<ProfileType[]> = await axios.get(`${BASE_URL}/users/${userId}/profiles`);
+    // console.log('getting all profiles:', response.data)
+    return response.data;
+} catch (e) {
+    console.log('error getting profiles', e);
+    throw e;
```

Comment:
we should not throw an error, thats the point of using try/catch

Reply:
It needs to be a throw error or else the Promise<UserType> starts screaming